app/bundles/DashboardBundle/Assets/js/dashboard.js

```diff
@@ -1,7 +1,82 @@
 //DashboardBundle
+Mautic.widgetUrl = '/s/dashboard/widget/';
```

Comment:
It should be:

`Mautic.widgetUrl = mauticBasePath + (mauticEnv == 'dev' ? '/index_dev.php' : '') + '/s/dashboard/widget/';`

Reply:
Oh! TY!